avogadro/core/residue.h

```diff
@@ -0,0 +1,87 @@
+/******************************************************************************
+
+  This source file is part of the Avogadro project.
+
+  Copyright 2018 Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#ifndef AVOGADRO_CORE_RESIDUE_H
+#define AVOGADRO_CORE_RESIDUE_H
+
+#include "avogadrocore.h"
+
+#include <map>
+#include <string>
+
+#include "array.h"
+#include "bond.h"
+#include "vector.h"
+
+namespace Avogadro {
+namespace Core {
+
+class Atom;
+class Molecule;
+
+/**
+ * @class Residue residue.h <avogadro/core/residue.h>
+ * @brief The Residue class represents a chemical residue, used commonly in the
+ * PDB format.
+ */
+class AVOGADROCORE_EXPORT Residue
+{
+public:
+  /** Type for atom name map. */
+  typedef std::map<std::string, Atom> AtomNameMap;
+
+  /** Creates a new, empty residue. */
+  Residue();
+  Residue(std::string& name);
+  Residue(std::string& name, Index& number);
+  Residue(std::string& name, Index& number, char& id);
+
+  Residue(const Residue& other);
+
+  Residue& operator=(Residue other);
+
+  virtual ~Residue();
+
+  std::string residueName();
+  void setResidueName(std::string& name);
+
+  Index residueId();
+  void setResidueId(Index& number);
+
+  char chainId();
+  void setChainId(char& id);
+
+  /** Adds an atom to the residue class */
+  void addResidueAtom(std::string& name, Atom& atom);
+
+  /** Returns a vector of atoms in the */
```

Comment:
Should be ``@return a vector of atoms in ``` ***???***
(last part of comment is missing)

Reply:
Apologies :( Somehow it went unnoticed.